.gitignore

```diff
@@ -9,4 +9,5 @@
 
 # OS metadata
 .DS_Store
-Thumbs.db
\ No newline at end of file
+Thumbs.db
+content/featured_background - Copy.png
```

Comment:
This is not a file that anyone else would have on their system, and probably isn't a file you'll regularly have on your system either, so I'd suggest we don't put it in `.gitignore`.

Reply:
Done.